grasping_geometric_primitives:
#####     PRIMITIVES    #####
 - name: gripper_r_frame
   type: frame 
   frame_id: gripper_r_base
   visible: true
   color: [0.0, 0.5, 0.0, 1.0]
   parameters: [0.0, 0.0, 0.15, 0.0, 0.0, 0.0]
 - name: gripper_l_frame
   type: frame 
   frame_id: gripper_l_base
   visible: true
   color: [0.0, 0.5, 0.0, 1.0]
   parameters: [0.0, 0.0, 0.15, 0.0, 0.0, 0.0]
 - name: pre_grasp_r_frame
   type: frame 
   frame_id: yumi_base_link
   visible: true
   color: [0.0, 0.5, 0.0, 1.0]
   parameters: [0.35, -0.15, 0.065, -1.57, 0.0, 1.57]
 - name: pre_grasp_l_frame
   type: frame 
   frame_id: yumi_base_link
   visible: true
   color: [0.0, 0.5, 0.0, 1.0]
   parameters: [0.35, 0.15, 0.065, 1.57, 0.0, 1.57]

#####     TASKS    #####
pre_grasp_tasks:
 - name: pre_grasp_r_track
   priority: 2
   visible: true
   active: true
   monitored: true
   def_params: ['TDefTracking', 'frame', 'frame', 'gripper_r_frame = pre_grasp_r_frame', '0.1']
   dyn_params: ['TDynPD', '1.0', '2.0']
#   def_params: ['TDefTracking', 'frame', 'frame', 'gripper_r_frame = pre_grasp_r_frame', '0.1', '0.5']
#   dyn_params: ['TDynPD', '10.0', '10.0', '10.0', '2.0', '2.0', '2.0', '0.05', '0.05', '0.05', '0.5', '0.5', '0.5']
 - name: pre_grasp_l_track
   priority: 2
   visible: true
   active: true
   monitored: true
   def_params: ['TDefTracking', 'frame', 'frame', 'gripper_l_frame = pre_grasp_l_frame', '0.1']
   dyn_params: ['TDynPD', '1.0', '2.0']
#   def_params: ['TDefTracking', 'frame', 'frame', 'gripper_l_frame = pre_grasp_l_frame', '0.1', '0.5']
#   dyn_params: ['TDynPD', '10.0', '10.0', '10.0', '2.0', '2.0', '2.0', '0.05', '0.05', '0.05', '0.5', '0.5', '0.5']
# - name: neutral pose
#   priority: 3
#   visible: true
#   active: true
#   monitored: false
#   def_params: ['TDefFullPose']
#   dyn_params: ['TDynPD', '2.0', '0.5']
   
grasp_tasks:
 - name: grasp_r_track
   priority: 2
   visible: true
   active: true
   monitored: true
   def_params: ['TDefTracking', 'frame', 'frame', 'gripper_r_frame = grasp_r_frame', '0.1']
   dyn_params: ['TDynPD', '0.5', '1.5']
 - name: grasp_l_track
   priority: 2
   visible: true
   active: true
   monitored: true
   def_params: ['TDefTracking', 'frame', 'frame', 'gripper_l_frame = grasp_l_frame', '0.1']
   dyn_params: ['TDynPD', '0.5', '1.5']
# - name: neutral pose
#   priority: 3
#   visible: true
#   active: true
#   monitored: false
#   def_params: ['TDefFullPose']
#   dyn_params: ['TDynPD', '2.0', '0.5']
   

#   dyn_params: ['TDynPD', '1.0', '2.0']
   
joint_config_tasks:
 - name: joint_config_start
   priority: 2
   visible: true
   active: true
   monitored: true
   def_params: ['TDefFullPose', '-1.0', '-1.4', '1.7', '-0.2', '0.0', '1.0', '1.0', '1.0', '-1.4', '-1.7', '-0.2', '0.0', '1.0', '-1.0']
   dyn_params: ['TDynPD', '0.5', '1.5']

neutral_config_tasks:
 - name: neutral_config
   priority: 3
   visible: true
   active: true
   monitored: true
   def_params: ['TDefFullPose', '-1.0', '-1.4', '1.7', '-0.2', '0.0', '1.0', '1.0', '1.0', '-1.4', '-1.7', '-0.2', '0.0', '1.0', '-1.0']
   dyn_params: ['TDynPD', '0.5', '1.5']
